src/test/resources/file_tls.properties

```diff
@@ -0,0 +1,3 @@
+alias=keycert
```

Comment:
Maybe I missed it, but I think this file isn't being used in a test right now. It would be great it your could add one, to ensure the file loading still works.

Reply:
You're right, I was in the process of adding an additional test (and support for file:// prefix). Commit added.